add-link

```diff
@@ -40,3 +40,9 @@ http://eternalstarculture.com/intern/home.php
 https://tinyurl.com/4cbf4wcp
 https://ig-feedbackcenter.com/form/983462456879/
 https://telifhakki-yardim.com/iletisim/439275610238
+https://supportlivecontact.com/portal/42534950324/pSkdQrxZkq.php
+http://facebooksupports.com
```

Comment:
Partially alive... in tor it's redirect to `https://supportlivecontact.com/`

On my own network I can't find any IP A or AAAA

It's not taken down, and it is running on it's own DNS server which rises the risk for reactivation... Would keep this one.

```
drill -T facebooksupports.com
.       518400  IN      NS      c.root-servers.net.
com.    172800  IN      NS      m.gtld-servers.net.
facebooksupports.com.   172800  IN      NS      ns1.facebooksupports.com.
facebooksupports.com.   172800  IN      NS      ns2.facebooksupports.com.

rill -t a facebooksupports.com
;; ->>HEADER<<- opcode: QUERY, rcode: SERVFAIL, id: 58001
;; flags: qr rd ra ; QUERY: 1, ANSWER: 0, AUTHORITY: 0, ADDITIONAL: 0 
;; QUESTION SECTION:
;; facebooksupports.com.        IN      A

;; ANSWER SECTION:

;; AUTHORITY SECTION:

;; ADDITIONAL SECTION:

```